### ISSUE TITLE
Version incompatibility between Node.js and npm in the npm rootfs image

### ISSUE THREAD

**Issue by thazhemadam** [JuliaCI/rootfs-images#219]
There seems to be a version incompatibility between the versions of Node.js (`v10.24.0`) and npm (`v5.8.0`) in the npm rootfs image published with the latest release.
This discrepancy can also be confirmed looking at the compatibility chart available at https://nodejs.org/en/download/releases.
These releases are also pretty old now, since the latest versions of Node.js and npm are `18.8.0` and `8.18.0` respectively.

Please note that I haven't tried using the npm rootfs image published as a part of the earlier releases, so I cannot comment on the usability of the same.

# MWE

```julia
julia --project test_rootfs.jl --url https://github.com/JuliaCI/rootfs-images/releases/download/v5.31/npm_linux.x86_64.tar.gz --treehash 893af31cd363fc852d65f383bf458193ec2d5d45
```

# Error
```julia
~/rootfs-images main ❯ julia --project test_rootfs.jl --url https://github.com/JuliaCI/rootfs-images/releases/download/v5.31/npm_linux.x86_64.tar.gz --treehash 893af31cd363fc852d65f383bf458193ec2d5d45
  Activating environment at `~/rootfs-images/Project.toml`
[ Info: Artifact did not exist locally, downloading
[ Info: Unpacking /tmp/jl_KA6GLu-download.gz into /home/zephyr/.julia/artifacts/jl_iYrqbq...
  Downloaded artifact: alpine-rootfs
juliaci@legion:/build$ npm -l
npm WARN npm npm does not support Node.js v10.24.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
...
```

**Comment by DilumAluthge** [comment]
This should be fixed by upgrading to Debian Bullseye (#222).

**Comment by DilumAluthge** [comment]
I think this should be fixed by the combination of #220, #222, and #221.

@thazhemadam Can you try v5.34 and see if the issue has been resolved?

**Comment by thazhemadam** [comment]
I can confirm that the issue seems to have been resolved on `v5.34` (and `v5.35`).

## `v5.34`
```bash
~/rootfs-images main ❯ julia --project test_rootfs.jl --url https://github.com/JuliaCI/rootfs-images/releases/download/v5.34/npm_linux.x86_64.tar.gz --treehash 32f5d89ce6768783f6135d1da8c2e490a4cb91bc
  Activating environment at `~/rootfs-images/Project.toml`
  Downloaded artifact: alpine-rootfs
I have no name!@deepsea3:/build$ npm -l
I have no name!@deepsea3:/build$
```

## `v5.35`
```bash
~/rootfs-images main ❯ julia --project test_rootfs.jl --url https://github.com/JuliaCI/rootfs-images/releases/download/v5.35/npm_linux.x86_64.tar.gz --treehash 3bc608bb90c292716877048c60b2bfdc2321ddda
  Activating project at `~/rootfs-images`
I have no name!@deepsea3:/build$ npm -l
I have no name!@deepsea3:/build$
```

Closing this issue since it seems to have been resolved. Thanks!
